components/Panel/VotePanel/OoTypeIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import OptimisticOracle from "public/assets/icons/optimistic-oracle.svg";
+import styled from "styled-components";
+import { OracleTypeT } from "types";
+
+export function OoTypeIcon({ ooType }: { ooType: OracleTypeT | undefined }) {
+  if (!ooType) return null;
+
+  const displayName = getOracleTypeDisplayName();
+
+  function getOracleTypeDisplayName() {
+    switch (ooType) {
+      case "OptimisticOracle":
+        return "OO v1";
```

Comment:
I think it would be better to call this the full thing, not the reduced version. `Optimistic Oracle`, `Optimistic Oracle V2` and `Skinny Optimistic Oracle`. There is enough space in the UI to accommodate this.

<img width="545" alt="2022-11-23 at 16 36 10@2x" src="https://user-images.githubusercontent.com/12886084/203573624-112196cd-3c51-4c3d-b69d-ca3ee39d7d60.png">